exercises/F-logical-operators/exercise2.js

```diff
@@ -5,13 +5,39 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0){
+    return true;
+  }
+  return false;
+}
+
+function isBetween5and10(num){
+  if (num >= 5 && num <= 10){
+    return true;
+  }
+  return false;
+}
+
+function isShortName(name){
+  if (name.length <= 6){
+    return true;
+  }
+  return false;
+}
+
+function startsWithD(name){
+  if(name[0] === "D"){
```

Comment:
This is incorrect as it assumes the name parameter is an array and checks the first index. The parameter should be a string and so name.charAt(0)  === "D" would be more appropriate.